client/my-sites/upgrades/trials/start-trial/terms-of-service.jsx

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Do we need a new component here? Can't we reuse `client/my-sites/upgrades/checkout/terms-of-service.jsx`?